uplift_report.ipynb

```diff
@@ -690,15 +733,20 @@
       ]
     },
     {
+      "cell_type": "code",
       "metadata": {
         "colab_type": "code",
         "id": "2y91jjPVy1Ai",
         "colab": {}
       },
-      "cell_type": "code",
       "source": [
+        "deduplicated_attributions_df = attributions_df\n",
+        "\n",
+        "if use_deduplication:\n",
+        "  deduplicated_attributions_df = drop_duplicates_in_attributions(attributions_df, pd.Timedelta('1 minute'))\n",
```

Comment:
IMHO there's no need to create another reference to a dataframe here, I'd rather keep `attributions_df` and assign to it if `use_deduplication` was true